cmd/skaffold/app/cmd/delete.go

```diff
@@ -50,6 +51,11 @@ func doDelete(ctx context.Context, out io.Writer) error {
 		if err != nil {
 			return fmt.Errorf("getting manifests from hydration dir: %w", err)
 		}
-		return r.Cleanup(ctx, out, dryRun, manifests)
+
+		manifestListByConfig, err := r.Render(ctx, io.Discard, []graph.Artifact{}, false)
```

Comment:
don't you have to initialize the `graph.Artifact` correctly to get the correct render output?

Reply:
Do you mean passing the actually build outcome to this method? I might have missed something, but by my investigation we only need some identifier like kind, metadata.name for deleting kubernetes resources, we don't need the exact the same manifests used for deployment for deletion. We're doing similar thing in v1 as well [implementation](https://github.com/GoogleContainerTools/skaffold/blob/release/v1.37.2/pkg/skaffold/deploy/kubectl/kubectl.go#L493-L535). Without passing the build results, we are actually skipping override image tag, allowlist and something, that shouldn't change the deletion result, right?

Reply:
Got it. Then this is working the same as in `v1`.